bot.py

```diff
@@ -5,26 +5,69 @@
 
 client = commands.Bot(command_prefix = "$")
 
-conn = sqlite3.connect('users.db')
+conn = sqlite3.connect(':memory:')
 
 c = conn.cursor()
 
 c.execute("""CREATE TABLE IF NOT EXISTS users (
              irl,
-             discord
+             game,
+             username
              )""")
 
-@client.event
-async def on_ready():
-    print("Bot is ready.")
-    print("test")
+def start_bot():

```

Comment:
Can this be a class or separate file? Define all the functions into one place and then expose a `client_run` function or something